src/jit/codegencommon.cpp

```diff
@@ -10265,7 +10265,52 @@ void CodeGen::genCaptureFuncletPrologEpilogInfo()
 
 /*****************************************************************************
  *
- *  Generates code for an EH funclet prolog.
+ *  Generates code for an EH funclet prolog. WIP.
+ *
+ *  Funclets have the following incoming arguments:
+ *
+ *      catch/filter-handler: ebp+8 = CallerSP, ebp+12 = the exception object that was caught (see GT_CATCH_ARG)
```

Comment:
JIT code looks like this. It uses EAX for the Exception object, but the object itself seems to be in the stack.
```
G_M9939_IG08:
       8945EC       mov      gword ptr [ebp-14H], eax
       8B4DEC       mov      ecx, gword ptr [ebp-14H]
       894DF4       mov      gword ptr [ebp-0CH], ecx
       90           nop      
       8B4DF4       mov      ecx, gword ptr [ebp-0CH]
       8B45F4       mov      eax, gword ptr [ebp-0CH]
       8B00         mov      eax, dword ptr [eax]
       8B402C       mov      eax, dword ptr [eax+44]
       FF5010       call     gword ptr [eax+16]System.Exception:get_Message():ref:this
```
source code:
```
            catch (Exception e)
            {
                Console.WriteLine(e.Message);
            }
```